code/src/Appsinglepage.js

```diff
@@ -0,0 +1,80 @@
+import React, { useState } from 'react'
```

Comment:
A suggestion for naming the components is using PascalCase, so for example instead of Appsinglepage it should be AppSinglePage. You can read more about it here: 
[https://www.notion.so/Guidelines-for-how-to-write-good-code-6a88a79b525f407f9bb039ea047bb7d2](url)